js/bootstrap-datepicker.js

```diff
@@ -919,8 +919,8 @@
 				tooltip;
 			if (isNaN(year) || isNaN(month))
 				return;
-			this.picker.find('.datepicker-days thead .datepicker-switch')
```

Comment:
Oh ok, if you want you can create a separate pr for that change! Just so we don't mix different change sets